src/content/en/2022/fonts.md

```diff
@@ -1,22 +1,851 @@
+
+Our guess is that developers—or their customers—really dislike seeing a flash of fallback fonts. Using `font-display: block` is an easy "fix" for that problem. However, there is a better solution on the horizon. In the near future you can use CSS font metric overrides to tweak your fallback fonts to approximate the metrics of your web fonts. This will reduce the jarring reflow of text when a fallback font is swapped with a web font.
+
+The CSS `ascent-override`, `descent-override`, `line-gap-override`, and `size-adjust` descriptors go into the `@font-face` rule and can be used to override the metrics in any font. You can use these descriptors with `local()` to [create a customized fallback](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/ascent-override#overriding_metrics_of_a_fallback_font) font that roughly matches your web font—hey, finally a good use for <a hreflang="en" href="https://bramstein.com/writing/web-font-anti-patterns-local-fonts.html">`local()`</a>.
+
+{{ figure_markup(
+  image="css-font-metrics-override-usage.png",
+  caption="CSS font metrics override usage.",
+  description="Column chart showing `ascent-override` is used on 0.11% of desktop and 0.20% mobile pages, `descent-override` on 0.07% and 0.13% respectively, `line-gap-override` on 0.07% and 0.13%, and finally `size-adjust` on 0.05% of desktop and 0.13% of mobile pages.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQF6OH_2-0apcFzjE-iHSQNuZqp9DtM7udIeOzPSOSMM-Pf6KdTnRwAclX9QPZF1vNNgu6acZvqoN5b/pubchart?oid=2058878625&format=interactive",
+  sheets_gid="241586017",
+  sql_file="font_metric_override_usage.sql"
+  )
+}}
+
+These `@font-face` descriptors are very new, but are already seeing some use. To make them even more useful developers need two things:
+
+1. A set of consistent fallback fonts that are available in all browsers and on all platforms. They could even be variable fonts. Imagine the possibilities.
+2. Tools to automatically match fonts by tweaking its size and metrics. Doing this by hand is very time-consuming, so a tool is a must. This is not intended as a replacement for the web font, but merely as a temporary fallback while the web fonts are loading (or as a standin if the fonts don't load or the browser is very old).
+
+We're slowly getting there with tools such as the <a hreflang="en" href="https://meowni.ca/font-style-matcher/">Font Style Matcher</a>, but unfortunately, fallback fonts are still very much platform dependent.
```

Comment:
Could also mention this one which calcs those new overrides for you: https://www.industrialempathy.com/perfect-ish-font-fallback/?font=Montserrat

Reply:
Oh, I didn't know that one. Thanks, I've added it.